roles/create-identities/tests/create_idm.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Modeled after this, could we create a more general playbook at the top level playbooks directory (see `load_infra.yml` for an example) and add it to the README? I would like to see us generate a set of playbooks that a fit for purpose and easy to adopt/use with other management tooling, such as Tower.